README.md

```diff
@@ -36,6 +36,10 @@ Please consider making a tax-deductible [donation](https://numfocus.org/donate-t
   >
 </a>
 
+## Public API
+
+Our public API is documented [in the API section of these docs](https://anndata.readthedocs.io/en/stable/api.html).  We cannot guarantee the stability of our internal APIs, whether it's the location of a function, its arguments, or something else.  In other words, we do not officially support (or encourage users to do) something like `from anndata._core import AnnData` as `_core` is both not documented and [contains a leading underscore](https://peps.python.org/pep-0008/#public-and-internal-interfaces).  However, we are aware that [many users do use these internal APIs](https://github.com/search?q=%22anndata._io%22&type=code) and thus encourage them to open an [issue](https://github.com/scverse/anndata/issues/new/choose) or migrate to the public API.  That is, if something is missing from out public API as documented, for example a feature you wish to be exported publicly, please open an issue.
```

Comment:
please

- maximally one sentence per source line
- no inline links
- PEP links are written ``{pep}`number` ``

Reply:
> PEP links are written {pep}`number` 

Any way to get the specific section?  I want people to go right there.  Also any docs about this stuff you could link to would be helpful.  I have trouble navigating the myst (I think that's where this comes from?) docs.

Reply:
Since this is the README, if we want this to be readable on the github page, we can’t use that stuff anyway sorry for the misdirection (we could use it if we used rST instead of MyST …)

For the record, you can use anchors: ``{pep}`number#anchor` ``

Reply:
> For the record, you can use anchors: {pep}`number#anchor` 

Cool, I suspected as much but wasn't sure.  Tough that I don't see much in the docs about it... https://myst-parser.readthedocs.io/en/latest/search.html?q=pep

Reply:
Maybe it's an rst thing? Would be great to understand more deeply